src/core/libmaven/mavenparameters.cpp

```diff
@@ -131,15 +152,38 @@ std::map<string, string>& MavenParameters::getSettings()
 
 void  MavenParameters::setPeakDetectionSettings(const char* key, const char* value)
 {
+    if(key[0] == '\0' || value[0] == '\0')
+        return;
+
     mavenSettings[const_cast<char*>(key)] = const_cast<char*>(value);
 
 
     if(strcmp(key, "automatedDetection") == 0 )
         processAllSlices = atof(value);
 
-//     TODO: replace with massCutoffMerge. remove ppmMerge from default_settings_xml as well. and regenerate settings.h 
-//     if(strcmp(key, "ppmStep") == 0 )
-//         ppmMerge = atof(value);
+
+     if(strcmp(key, "massCutoffType") == 0 ) {
```

Comment:
Aren't these 3 separate variables?

Reply:
no. it tells whether the type is 'ppm' or 'mDa'

Reply:
Got it